comment/tests/test_api/test_views.py

```diff
@@ -511,22 +512,29 @@ def test_flag_to_previous_comments(self):
         comment.refresh_from_db()
         self.assertEqual(comment.flag.count, 1)
 
+    @patch.object(settings, 'COMMENT_FLAGS_ALLOWED', 1)
     def test_is_flagged_value(self):
         # test is_flagged property
         comment = self.create_comment(self.content_object_1)
         data = self.flag_data
         url = self.get_url(comment.id)
-        settings.COMMENT_FLAGS_ALLOWED = 1
         response = self.client.post(url, data=data)
+
         self.assertEqual(response.status_code, 200)
-        self.assertFalse(response.json()['is_flagged'])
+        self.assertIs(False, response.json()['is_flagged'])
```

Comment:
As previously discussed, `is_flagged` is a boolean value and assertFalse can be used directly.

Reply:
`assertIs` as mentioned in the documentation is more powerful for this type of test. It also provides a more helpful message when debugging.

Reply:
```sh
 $ python                                   
Python 3.7.5 (default, Nov  7 2019, 10:50:52) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import unittest
>>> test = unittest.TestCase()
>>> test.assertIs.__doc__
'Just like self.assertTrue(a is b), but with a nicer default message.'
```

Reply:
> `assertIs` as mentioned in the documentation is more powerful for this type of test. It also provides a more helpful message when debugging.

In fact the debug message for assertTrue is pretty much descriptive: `False is not True` do we really need more info about this? 😄

Reply:
As I know, you are a big fan of less typing 🙄

Reply:
I peeked into the official django docs, and found that they also prefer to use `assertIs` over `assertTrue`.

> As I know, you are a big fan of less typing roll_eyes

I don't want to get distracted by this candy :laughing: